README.rst

```diff
@@ -4,6 +4,14 @@ CRATE Django
 
 A django database backend for Crate.IO
 
+
+Install
+=======
+
+git clone https://github.com/mfelsche/crate-django.git
+cd crate-django
+python setup.py install
```

Comment:
for development purposes the way to go is running:

```
python bootrstap.py
bin/buildout
```

preferrably in a virtualenv.
This will create a sandboxed environment suitable for development.